docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -179,19 +192,37 @@ The AI Assistant uses functions to include relevant context in the chat conversa
 
 The following table lists available functions:
 
+//TODO: Resolve this question:
+**REVIEWERS**: Is the list of functions here correct? Only a subset are actually shown in the UI. None of the apm functions are shown in the UI right now.
+
+//TODO: Resolve this question:
+**REVIEWERS**: Should we replace this table with the UI instructions for showing the list of functions (like we did in serverless)?
```

Comment:
I'm fine with leaving them here, but we should consider them an implementation detail while they're still in beta. They can easily change or be removed entirely (like we did with the lens function).